web/styles/inbox.css

```diff
@@ -472,7 +472,7 @@
                 box-shadow: 0 0 0 2px var(--color-outline-focus);
             }
 
-            & i {
+            .zulip-icon {
```

Comment:
zulip-icon is not a much better selector -- it still will initially match every Zulip icon in the app, aka eventually basically all of them. The right fix is to adjust the template to add a class that we can reference.